### PR TITLE
Modal how to play

### PR DESCRIPTION
**Pull request by hassansaleem194**
- Added how to play modal text and CSS
- Added test for modal

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -8,12 +9,55 @@
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
     <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
+<<<<<<< HEAD
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
```
Comment (BethLambertMatthews) [inline app/views/layouts/application.html.erb:12]: Duplicate copy of the Bootstrap stylesheet link - could delete?

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -8,12 +9,55 @@
...
+<!-- Button to Open the Modal -->
+
+<!-- The Modal -->
+    <div class="modal" id="myModal">
+      <div class="modal-dialog">
+        <div class="modal-content">
+
+          <!-- Modal Header -->
+          <div class="modal-header">
+            <h4 class= "how-to-text"> Here's how to play Buger Hunt</h4>
```
Comment (BethLambertMatthews) [inline app/views/layouts/application.html.erb:28]: Typo Buger

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -8,12 +9,55 @@
...
+          <div class="modal-body">
+            <p class= 'how-to-text'> Many years ago, some really smart people or person discovered the existence of electricity.
+            A whole bunch of years later, some more smart people or person harnessed that power that great power to build
+            a device called a computer. Fast forward again and now we have the ability to use their efforts of our forefathers
+            to build spectacular video games. You're welcome. </p>
+            <p class= 'how-to-text'> Burger Hunt is a pretty straight forward game. Read the text to dive into a well written and not at all
+            hashed together story. You will be presented with choices and puzzles along the way. Use your intellect to navigate
+            your way through trecherous streets to reach your destination: a burger restaurant. We encourage you to be curious
+            and think outstide the box, but not too far outside, it's scary out there. </p>
+            <p class= 'how-to-text'> Good luck and stay fresh.</p>
```
Comment (BethLambertMatthews) [inline app/views/layouts/application.html.erb:42]: Nice!

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -8,12 +9,55 @@
...
+          <div class="modal-footer">
+            <button type="button" class="btn btn-danger" data-dismiss="modal">Close</button>
+          </div>
+
+        </div>
+      </div>
+    </div>
     <%= yield %>
+
+    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
```
Comment (BethLambertMatthews) [inline app/views/layouts/application.html.erb:55]: Might be duplicates with the lines below 62 onwards?

File: spec/features/how_to_play_spec.rb [hunk 5]
```diff
@@ -0,0 +1,7 @@
+RSpec.feature "How to play", type: :feature do
+  scenario "Players can access the how to play modal" do
+    visit ('/')
+    click_button 'How to play'
+    expect(page).to have_content 'Here\'s how to play Buger Hunt'
```
Comment (BethLambertMatthews) [inline spec/features/how_to_play_spec.rb:5]: This will need to be updated when the typo is fixed :-)

**Review by BethLambertMatthews — CHANGES_REQUESTED** [review]
Looks great - couple of duplicates and a typo to be fixed!

**Review by BethLambertMatthews — APPROVED** [review]

**Review by troysnowden — APPROVED** [review]
LGTM!

**Pull request merged**